src/api/communities.ts

```diff
@@ -32,4 +32,11 @@ async function createCommunity(createCommunityDto: CreateCommunityDto) {
   );
 }
 
-export { search, getAllCommunities, createCommunity };
+async function getCommunityByName(communityName: string): Promise<Community> {
+  const CommunityResponse = await api.get("/communities/" + communityName, {
```

Comment:
1. you're passing an empty object in the second parameter of your .get() which is redundant as a GET doesn't take a body and it doesn't appear that you need to pass any params in this case

2. possibly stick to string interpolation when concatenating strings, more readable
example:
api.get(\`/communities/${communityName}\`)

using backticks ` instead of ' or " allows you to pass dynamic values by wrapping them with ${}